bot/cogs/memes_cog.py

```diff
@@ -21,5 +27,39 @@ async def bubblewrap(self, ctx):
 
         await ctx.send(msg)
 
+    @commands.command()
+    async def spongebob(self, ctx, *, args):
+
+        '''
+        Spongebob Text
+        '''
+        seed(time())
+        args = args.replace('"', "'")
+    
+        result = ''
+        for i in args:
+            helper = randint(0, 100)
+            
+            if helper > 60:
+                result += str(i).upper()
+            else:
+                result += str(i).lower()
+
+        try:
+            embed = discord.Embed(title="SpOnGeBoB", color=Colors.ClemsonOrange)
+            result2 = ''
+            if len(result) >= 1024:
+                result2 = result[1024:len(result)]
+                result = result[:1023]
+            embed.add_field(name="TeXt", value=result, inline=False)
+            if result2:
+                print("RESULT2\n\n\n")
+                embed.add_field(name="tExT", value=result2, inline=False)
+        except Exception as e:
```

Comment:
We dont need this except block here, we can let an exception bubble up to the top so it can be correctly logged as well as sent